src/riscv/etherlink/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[workspace]
+members = ["kernel"]
```

Comment:
What else do you intend to add to this workspace?

Reply:
A crate for benchmarking, similar to jstz.